packages/typescript-estree/src/create-program/createWatchProgram.ts

```diff
@@ -175,7 +175,7 @@ function getProgramsForProjects(
 
       updatedProgram =
         updatedProgram ?? existingWatch.getProgram().getProgram();
-      // sets parent pointers in source files
+      // sets parent pointers in source files when module === none
```

Comment:
```suggestion
      // sets parent pointers in source files
```

I thought this was required regardless, not just when in no module mode?

Reply:
This seems to be no longer needed (at least for ts 3.7) if module is != none